rfc/2022/2022-07-13 - Platform Wide Documentation vs Component Specific Documentation Locations.md

```diff
@@ -0,0 +1,102 @@
+* VFS teams will find it easier to document information about downstream systems external to the VA. They can either create documentation collections for these systems or import the documentation directly from these teams if they already exist.
+* New VFS and Platform team members will more easily be able to learn about the software projects that their team maintains.
+* VFS and Platform team members can easily find and discover documentation about any VFS and Platform entity. The Console UI has interfaces for searching through all projects with documentation across the entire catalog. Each collection of documentation also has a search interface that will confine searches to specific projects or assets.
+* Each component’s source code and documentation will be easier to find and less likely to be lost than if they were stored in diverse locations.
+* With connections between component and their documentation better maintained, it will be easier to care for each software component, its documentation, and systems that depend upon the component.
+* With each component's documentation stored close to its source code, it will be easier to run automated checks to see if each component's documentation is up to date. Keeping the documentation with the source code also ensures that we're following VA policy to make all documentation publicly accessible.
+* Confidence in having access to a single source of truth to ensure that engineers have access to the most relevant, up-to-date information
+
+## Risks
+
+* A risk of this strategy is that Platform-wide documentation intended for all VFS developers could be mistakenly added to the Console instead of the Platform Website, making it harder to find. Platform and VFS Engineers will need clear instructions to avoid this.
+* If the Console UI system is retired, all dispersed component-specific documentation loses its central location for being found. This scenario would be a sub-optimal situation but still better than storing documentation in a central system separate from the source code of assets. Elimination of such a system may permanently separate software from the documentation.
+
+## Alternatives
+
+* We could drive people to add documentation for their components/projects to the Platform Website. The risk associated with this alternative is that documentation may become separated from source code and add irrelevant documentation to some VFS developer audiences.
+* We could encourage developers to use a system like Confluence for all component-specific technical documentation. But that may incur additional licensing costs and the possibility that all documentation is lost if Confluence, or a similar system, is retired. Also, VFS team members do not always have access to Confluence.
+
+**Questions from Platform Content team to consider:**
```

Comment:
These questions were added in as if they were part of the RFC, but I think they need to be removed if they've been addressed?

Reply:
I have removed them. Thanks for noticing @sdk-irl.